npm/webpack-dev-server/src/plugin.ts

```diff
@@ -85,12 +86,10 @@ export default class CypressCTOptionsPlugin {
       },
     )
 
-    compiler.hooks.afterEmit.tap(
+    compiler.hooks.done.tap(
       'CypressCTOptionsPlugin',
-      (compilation) => {
-        if (!compilation.getStats().hasErrors()) {
-          this.devServerEvents.emit('dev-server:compile:success')
-        }
+      () => {
+        this.refreshCompile = true
```

Comment:
Is it correct to trigger this even if there is errors? I notice we deleted `compilation.getStats().hasErrors()`

Reply:
I truly have no idea. 
The UX is weird when compilation fails as the tests look green but an error shows in the AUT panel.

### webpack in master
the command log pannel seems untouched ebven if the build failed
<img width="1433" alt="Webpack error on master" src="https://user-images.githubusercontent.com/5592465/133839210-45b0d057-d4b9-4a0f-add4-4daf631931ee.png">

### webpack in this PR
command log is sending a cryptic set of errors
<img width="1389" alt="Webpack error on this PR" src="https://user-images.githubusercontent.com/5592465/133839242-e1ead781-79f8-4f9a-a534-dc4b07fbab6e.png">

Reply:
reversed this change in c8f5816